coalib/bearlib/abstractions/Linter.py

```diff
@@ -711,6 +714,8 @@ class GlobalLinterMeta(type(LinterBase), type(GlobalBear)):
 
     class GlobalLinterBase(LinterBase, GlobalBear, metaclass=GlobalLinterMeta):
 
+        LinterParent.register(LinterBase)
```

Comment:
hm... maybe put it into `Linter.py` then? Hopefully nobody needs to import like this then...
```
from coalib.abstractions.Linter import linter, Linter
```